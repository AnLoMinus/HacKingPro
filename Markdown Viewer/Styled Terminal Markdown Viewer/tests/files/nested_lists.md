# Lists

- l1
    - l2
    - l3
        - l4
    - l5
- l2


> with sep2



* l1
    * l2
    * l3
        * l4
    * l5
* l2


