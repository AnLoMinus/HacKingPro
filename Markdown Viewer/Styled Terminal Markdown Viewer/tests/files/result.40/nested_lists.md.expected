
[38;5;59m[0m[38;5;209mLists[0m
[38;5;188m    [38;5;209m- [0ml1
[38;5;188m        [38;5;74m- [0ml2[0m
[38;5;188m        [38;5;74m- [0ml3
[38;5;188m            [38;5;32m- [0ml4[0m
[38;5;188m        [38;5;74m- [0ml5[0m
[38;5;188m    [38;5;209m- [0ml2[0m
[38;5;209m|[0m[38;5;188m   with sep2[0m
[38;5;188m    [38;5;209m- [0ml1
[38;5;188m        [38;5;74m- [0ml2[0m
[38;5;188m        [38;5;74m- [0ml3
[38;5;188m            [38;5;32m- [0ml4[0m
[38;5;188m        [38;5;74m- [0ml5[0m
[38;5;188m    [38;5;209m- [0ml2[0m


